example_project.json

```diff
@@ -1,421 +1,580 @@
+				-78.85038975191084,
+				42.81803226424341
+			  ],
+			  [
+				-78.850630729414,
+				42.81857888627522
+			  ],
+			  [
+				-78.85025301954843,
+				42.81866847653532
+			  ],
+			  [
+				-78.8500120420453,
+				42.81812185529549
+			  ]
+			]
+		  ]
+		}
+	  },
+	  {
+		"type": "Feature",
+		"properties": {
+		  "id": "3",
+		  "name": "Restaurant 10",
+		  "type": "Building",
+		  "footprint_area": 41877,
+	  	  "floor_area": 125631,
+		  "number_of_stories": 3,
+		  "building_type": "Food service"
+		},
+		"geometry": {
+		  "type": "Polygon",
+		  "coordinates": [
+			[
+			  [
+				-78.84962224800356,
+				42.81329273502644
+			  ],
+			  [
+				-78.84929833482822,
+				42.81337083838241
+			  ],
+			  [
+				-78.84983265832118,
+				42.814563298664666
+			  ],
+			  [
+				-78.85015657149653,
+				42.81448519681467
+			  ],
+			  [
+				-78.84962224800356,
+				42.81329273502644
+			  ]
+			]
+		  ]
+		}
+	  },
+	  {
+		"type": "Feature",
+		"properties": {
+		  "id": "4",
+		  "name": "Restaurant 12",
+		  "type": "Building",
+		  "footprint_area": 10541,
+		  "floor_area": 31623,
+		  "number_of_stories": 3,
+		  "building_type": "Food service"
+		},
+		"geometry": {
+		  "type": "Polygon",
+		  "coordinates": [
+			[
+			  [
+				-78.84907318596754,
+				42.81342719667407
+			  ],
+			  [
+				-78.84862090048105,
+				42.81353625345659
+			  ],
+			  [
+				-78.84871721918239,
+				42.813751210926796
+			  ],
+			  [
+				-78.84916950466888,
+				42.81364215452331
+			  ],
+			  [
+				-78.84907318596754,
+				42.81342719667407
+			  ]
+			]
+		  ]
+		}
+	  },
+	  {
+		"type": "Feature",
+		"properties": {
+		  "id": "5",
+		  "name": "Restaurant 14",
+		  "type": "Building",
+		  "footprint_area": 8804,
+		  "floor_area": 8804,
+		  "number_of_stories": 1,
+		  "building_type": "Food service"
+		},
+		"geometry": {
+		  "type": "Polygon",
+		  "coordinates": [
+			[
+			  [
+				-78.84809175426629,
+				42.81367038997507
+			  ],
+			  [
+				-78.84848670778973,
+				42.81357515750889
+			  ],
+			  [
+				-78.84857883872144,
+				42.81378076888831
+			  ],
+			  [
+				-78.84818388519801,
+				42.81387600103781
+			  ],
+			  [
+				-78.84809175426629,
+				42.81367038997507
+			  ]
+			]
+		  ]
+		}
+	  },
+	  {
+		"type": "Feature",
+		"properties": {
+		  "id": "6",
+		  "name": "Restaurant 15",
+		  "type": "Building",
+		  "footprint_area": 10689,
+		  "floor_area": 10689,
+		  "number_of_stories": 1,
+		  "building_type": "Food service"
+		},
+		"geometry": {
+		  "type": "Polygon",
+		  "coordinates": [
+			[
+			  [
+				-78.84846106738529,
+				42.814495803077364
+			  ],
+			  [
+				-78.8486903952376,
+				42.81444050756261
+			  ],
+			  [
+				-78.8484977578349,
+				42.81401059666683
+			  ],
+			  [
+				-78.84826842998261,
+				42.81406589256599
+			  ],
+			  [
+				-78.84846106738529,
+				42.814495803077364
+			  ]
+			]
+		  ]
+		}
+	  },
+	  {
+		"type": "Feature",
+		"properties": {
+		  "id": "7",
```

Comment:
@vtnate The previous Feature file had the detailed model filename for some features which we should retain

Reply:
Thanks for the assist @tanushree04, you are a badass.